P/Pfapack/build_tarballs.jl

```diff
@@ -0,0 +1,62 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Pfapack"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/xrq-phys/Pfapack.git", "0c71536b9641a0c8f4da67b373e3d4d5514561ab")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd Pfapack
+
+cmake fortran \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release
+make -j${nproc} VERBOSE=1
+make install
+
+# Manual conversion from static to dynamic lib.
+# Avoid linking libgfortran (these .f files make no reference to Fortran libs.)
+cc -shared -Wl,-force_load,${prefix}/lib/libpfapack.a \
+    -o ${prefix}/lib/libpfapack.${dlext} \
+    -L${libdir} -lblastrampoline -lm \
+
+# Copy license file
+install_license LapackLicence
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("x86_64", "linux"; libc="musl"),
+    Platform("armv7l", "linux"; libc="glibc"),
+    Platform("x86_64", "windows"),
+    Platform("x86_64", "macos"),
+    Platform("x86_64", "linux"; libc="glibc"),
+    Platform("aarch64", "linux"; libc="glibc"),
+    Platform("aarch64", "macos"),
+    Platform("x86_64", "freebsd")
+]
```

Comment:
Out of curiosity: this is bizarre set of platforms, what's the problem with the others?